bin/collator/src/local/service.rs

```diff
@@ -290,6 +319,11 @@ pub fn start_node(config: Configuration) -> Result<TaskManager, ServiceError> {
         let client = client.clone();
         let network = network.clone();
         let transaction_pool = transaction_pool.clone();
+        let rpc_config = crate::rpc::EvmTracingConfig {
+            tracing_requesters,
+            trace_filter_max_count: tracing_config.ethapi_trace_max_count,
+            enable_txpool: true,
```

Comment:
Shouldn't this be `enable_txpool: ethapi_cmd.contains(&EthApiCmd::TxPool),` like in the code for parachain service?

Reply:
I believe it's ok to enable it by default for `--dev` node, but your proposal works too.